component/componenthelper/component.go

```diff
@@ -31,7 +31,7 @@ func (f StartFunc) Start(ctx context.Context, host component.Host) error {
 // ShutdownFunc specifies the function invoked when the component.Component is being shutdown.
 type ShutdownFunc func(context.Context) error
 
-// Shutdown calls f(ctx, host).
+// Shutdown calls f(ctx).
```

Comment:
Please don't document the implementation details in godoc, instead explain the behavior. In this case it's very arbitrary because it'll be something like "Shutdown shuts down".

Reply:
Addressed this comment. Changed to "Shutdown shuts down the component.". Made a similar change for Start as well.

Reply:
@rakyll good call, comment was inspired by https://golang.org/src/net/http/server.go?s=64180:64240#L2058